packages/app/src/domain/vaccine/milestones-view.tsx

```diff
@@ -58,71 +58,87 @@ export function MilestonesView(props: MilestoneViewProps) {
         position="relative"
         css={css({ listStyleType: 'none' })}
       >
-        <ListItemFirst isExpanded={isExpanded}>
+        <ListItemFirst>
           <CircleSmall />
           <Text fontWeight="bold" m={0} pl={`calc(1rem)`}>
             2021
           </Text>
         </ListItemFirst>
 
-        {milestones.length > MAX_ITEMS_VISIBLE && !isExpanded && (
+        {milestones.length > MAX_ITEMS_VISIBLE && (
           <ListItem>
             <Box pl={`calc(1rem + ${CIRCLE_SIZE}px)`}>
-              <ExpandButton color="link" onClick={handleExpand}>
-                {siteText.milestones.toon_meer}
+              <ExpandButton
+                color="link"
+                onClick={handleExpand}
```

Comment:
nit; we could easily create an inline function here
```suggestion
                onClick={() => setIsExpanded(x => !x)}
```